packages/bulkapi/src/controllers/BulkApiWorker.ts

```diff
@@ -116,4 +117,50 @@ export class BulkApiWorker {
             }
         }
     }
+
+    ids = async (clientId: string, count: number): Promise<QueryReturnType<IdsResponse>> => {
```

Comment:
maybe `ids` is not completely obvious as a name. What about `reserveIDs` or `askForIDs` or `assignIDs`?

Reply:
`ids` is the name for this method given in the LionWeb spec, I agree it quite terse and we could do with a better name. But we need to change this in the spec first.